### ISSUE TITLE
Disable Update Prompt

### ISSUE THREAD

**Issue by Nexinity7** [thibauddavid/onewheelenhancer-issues#8]
Not sure if this is abandonware at this point, but as a new CBXR user this tweak rocks and I would love if there was a way to permanently disable the update firmware prompt. Thank you for this tweak!

**Comment by thibauddavid** [comment]
Hi @Nexinity7 , glad to see some users still use this. 
New firmwares aren't supposed to get downloaded when onewheelenhancer is installed.

Did you install it after firmware got downloaded ?
Are you using a sideloaded version, or did you install it through Cydia on a jailbroken phone ?

**Comment by Nexinity7** [comment]
I'm on a jailbroken 6S with the tweak installed through Zebra. I think i might have fat fingered a go button but nothing happened because i'm on hardware 4208 with very bad bluetooth lol, or if dependencies are removed it couldn't download anything. Nothing happened, was just wondering if there's a way to permanently disable all update prompts regardless.

**Comment by thibauddavid** [comment]
> Did you install it after firmware got downloaded ?

Did you ? I prevent firmwares to be downloaded, but maybe you install the tweak after, that would explain the behavior you encounter. If so, you can try deleting OW app, reinstalling it, and reinstall tweak using Zebra. Only then, reopen app, and firmware shouldn't ever be downloaded (thus you won't be prompted to install it)

Keep me in touch 👍

**Comment by Nexinity7** [comment]
No firmware was downloaded. Method of installation was 
1. Download OW from App Store
2. Download Tweak and respring 
3. Open OW
4. Connect in OW
5. Get prompted to update (everytime the board connects it'll get prompted to update)

i was just wondering if there's a way to permanently prevent any pop up prompting an update, it gets annoying but i am glad there's a later button

**Comment by thibauddavid** [comment]
Yeah but internally, app first downloads firmware, then displays popup to prompt user to upload it to board and install it.
So if you connected your board to app before installing tweak, firmware was most probably downloaded
